CMakeLists.txt

```diff
@@ -148,8 +148,8 @@ if(NOT MINIMAL_BUILD)
   # libcurl
   include(curl)
 
-  # civetweb
-  include(civetweb)
+  # cpp-httlib
```

Comment:
Just curious, why the switch to cpp-httplib?

Reply:
Civetweb caused lots of compilation problems in the past, and was in general a troublesome dependency for us in the past. This new http server is header-only and pretty lightweight, which better serves out purposes for now since we only implement `/healthz`.